patches/0001-LPS-89596-Cannot-Drag-Image-from-top-content-line-in.patch

```diff
@@ -25,5 +25,5 @@ index 1d4e036fc..a53a4c809 100644
  				return;
  
 -- 
-2.27.0
+2.24.3 (Apple Git-128)
```

Comment:
> Apple Git-128 

Funny they didn't use `iGit`

Reply:
And so necessary of them to call out that they've made their own little special build...

![snowflake](https://user-images.githubusercontent.com/7074/87039129-ff6e0900-c1ee-11ea-9bc8-c00fabbdab58.gif)